vita3k/gui/src/app_selector.cpp

```diff
@@ -457,20 +468,30 @@ void draw_app_selector(GuiState &gui, HostState &host) {
         break;
     }
 
+    const auto SELECT_SIZE = ImVec2(50.f * scal.x, 60.f * scal.y);
+    ImGui::SetWindowFontScale(2.f * scal.x);
+    ImGui::PushStyleVar(ImGuiStyleVar_SelectableTextAlign, ImVec2(0.5f, 0.5f));
+    if (current_scroll_pos > 0) {
+        ImGui::SetCursorPos(ImVec2(display_size.x - SELECT_SIZE.x - (5.f * scal.x), 48.f));
+        if ((ImGui::Selectable("^", false, ImGuiSelectableFlags_None, SELECT_SIZE)) || ImGui::IsKeyPressed(host.cfg.keyboard_leftstick_up) || ImGui::IsKeyPressed(host.cfg.keyboard_button_up))
+            scroll_type = 1;
+    }
     if (!gui.apps_list_opened.empty()) {
-        const auto SELECT_SIZE = ImVec2(50.f * scal.x, 60.f * scal.y);
-        ImGui::SetWindowFontScale(2.f * scal.x);
-        ImGui::PushStyleVar(ImGuiStyleVar_SelectableTextAlign, ImVec2(0.5f, 0.5f));
         ImGui::SetCursorPos(ImVec2(display_size.x - SELECT_SIZE.x - (5.f * scal.x), (display_size.y / 2.f) - (SELECT_SIZE.y / 2.f)));
         if ((ImGui::Selectable(">", false, ImGuiSelectableFlags_None, SELECT_SIZE)) || ImGui::IsKeyPressed(host.cfg.keyboard_button_r1) || ImGui::IsKeyPressed(host.cfg.keyboard_leftstick_right)) {
             last_time["start"] = 0;
             ++gui.current_app_selected;
             gui.live_area.app_selector = false;
             gui.live_area.live_area_screen = true;
         }
-        ImGui::PopStyleVar();
-        ImGui::SetWindowFontScale(1.f * scal.x);
     }
+    if (current_scroll_pos < max_scroll_pos) {
+        ImGui::SetCursorPos(ImVec2(display_size.x - SELECT_SIZE.x - (5.f * scal.x), display_size.y - SELECT_SIZE.y - (34.f * scal.y)));
+        if ((ImGui::Selectable("v", false, ImGuiSelectableFlags_None, SELECT_SIZE)) || ImGui::IsKeyPressed(host.cfg.keyboard_leftstick_down) || ImGui::IsKeyPressed(host.cfg.keyboard_button_down))
```

Comment:
```suggestion
        if ((ImGui::Selectable("v", false, ImGuiSelectableFlags_None, SELECT_SIZE))
            || ImGui::IsKeyPressed(host.cfg.keyboard_leftstick_down)
            || ImGui::IsKeyPressed(host.cfg.keyboard_button_down))
```
Split it over many lines?

Reply:
hmm yes but here is only 3 so is good ^^

Reply:
The line is 219 characters long. The line limit for most linters is 80. I think it would be nice to at least try to stay near the limit.

Fix if you like. We go way over elsewhere in the project.

Reply:
i see but so if is that need change alll othetr inside gui code lol i have set very some with little long line haha

Reply:
done for it, news line for separe selectable/keyboard press